org.freedesktop.Sdk.Extension.dart.yaml

```diff
@@ -0,0 +1,60 @@
+id: org.freedesktop.Sdk.Extension.dart
+branch: "22.08"
+runtime: org.freedesktop.Sdk
+build-extension: true
+sdk: org.freedesktop.Sdk
+runtime-version: "22.08"
+separate-locales: false
+appstream-compose: false
+modules:
+  - name: dart
+    buildsystem: simple
+    sources:
+      - type: archive
+        only-arches:
+          - aarch64
+        url: https://storage.googleapis.com/dart-archive/channels/stable/release/2.18.6/sdk/dartsdk-linux-arm64-release.zip
+        sha256: 61dbb462b48aee4f3184b6ecdd356632f39165ae8570fe77a62900a6444f702c
+        x-checker-data:
+          type: anitya
+          project-id: 14718
+          stable-only: true
+          url-template: https://storage.googleapis.com/dart-archive/channels/stable/release/$version/sdk/dartsdk-linux-arm64-release.zip
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://storage.googleapis.com/dart-archive/channels/stable/release/2.18.6/sdk/dartsdk-linux-x64-release.zip
+        sha256: 492c0e835203c4402e3d8291d12b53927f0300c8080aaf63a9113c204255a735
+        x-checker-data:
+          type: anitya
+          project-id: 14718
+          stable-only: true
+          url-template: https://storage.googleapis.com/dart-archive/channels/stable/release/$version/sdk/dartsdk-linux-x64-release.zip
+    build-commands:
+      - install -d /usr/lib/sdk/dart
+      - cp -rpv * /usr/lib/sdk/dart/
+  - name: scripts
+    modules:
+      - dart
+    sources:
+      - type: script
+        commands:
+          - export DARTROOT=/usr/lib/sdk/dart
+          - export PATH=$PATH:$DARTROOT/bin
+        dest-filename: enable.sh
```

Comment:
I'm not sure that this works as intended. I tried sourcing this enable script locally and it didn't add dart to the path.

Reply:
NVM. The extension is loaded automatically into the environment if it is installed. This wasn't clear to me as a new flatpak user.